src/components/NavBar/index.js

```diff
@@ -133,6 +144,21 @@ class NavBar extends Component {
   }
 }
 
+const FAQIcon = ({ intl }) => (
+  <a
+    target='_link'
+    href={intl.formatMessage(messages.url)}
+    style={{
+      margin: 'auto 0px auto 20px',
+      color: 'white',
+      fontSize: '1.47em',
+      fontWeight: 700,
+    }}
+  >
+    ?
```

Comment:
use https://material.io/icons/#ic_live_help for now until we can get one from designers